core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -0,0 +1,315 @@
+		this.stage = new Stage(stageViewport);
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		Table roster = new Table();
+		
+		Actor employeePane = createEmployeePane();
+		Actor employeeModel = createEmployeeModel();
+		
+		roster.add(employeeModel).top().spaceRight(75);
+		roster.add(employeePane).top().spaceLeft(75);
+		
+		roster.setTransform(true);
+		roster.setPosition(DESIGN_SCREEN_CENTER_X, DESIGN_SCREEN_CENTER_Y);
+		
+		// TODO: Corner Advisor		
+		employeePane.setPosition(DESIGN_WIDTH / 4 + ROSTER_OFFSET, DESIGN_HEIGHT / 6);
+		employeePane.setSize(350, 400);
+		roster.setPosition(ROSTER_OFFSET, DESIGN_HEIGHT / 2);
+		
+		stage.addActor(roster);
+		stage.addActor(employeePane);
+		
+	}
+	
+	public Company getCompanyContext()
+	{
+		return new Company() {
+			
+			@Override
+			public List<Employee> getEmployees()
+			{
+				Employee employee1 = new Employee("Jason Richards", "Manager", 14, 11,
+						10, 1600, 15.6f, 3.2f, 150);
+				Employee employee2 = new Employee("Janet Wilmore", "Product Creation", 8,
+						8, 10, 1000, 6.7f, 1.2f, 35);
+				
+				List<Employee> employees = new LinkedList<>();
+				employees.add(employee1);
+				employees.add(employee2);
+				
+				return employees;
+			}
+			
+		};
+	}
+	
+	/**
+	 * Creates the Employee Pane, a list of everyone employed to the business. Allows user
+	 * to select the "New Employee" option which expands another pane where you can hire
+	 * new employees.
+	 */
+	private Actor createEmployeePane()
+	{
+		String newEmp = "New Employee";
+		VerticalGroup employeeTable = new VerticalGroup();
+		ScrollPane employeeScrollPane = new ScrollPane(employeeTable, skin);
+		
+		for (Employee employee : getCompanyContext().getEmployees())
+		{
+			addEmployee(employee, employeeTable);
+		}
+		
+		newEmployee(newEmp, employeeTable);
+		return employeeScrollPane;
+	}
+	
+	private void addEmployee(Employee employee, VerticalGroup group)
+	{
+		Label label = new Label(employee.toString(), skin);
+		group.addActor(label);
+		label.setAlignment(Align.center);
+		label.addListener(new EmployeeListener(employee));
+	}
+	
+	private void newEmployee(String newEmp, VerticalGroup group)
+	{
+		Label label = new Label(newEmp, skin);
+		group.addActor(label);
+		label.setAlignment(Align.center);
+	}
+	
+	/**
+	 * Creates the Employee Model that displays the selected employees information
+	 * including Name, Pay, Income Preference, and morale.
+	 */
+	private Actor createEmployeePay()
+	{
+		employeePayTable = new Table(skin);
+		Label payLabel = new Label("Pay: $", skin);
+		Label suffix = new Label(" / hr", skin);
+		employeePayField = new TextField("", skin);
+		
+		employeePayField.setTextFieldListener(new TextField.TextFieldListener() {
+			
+			@Override
+			public void keyTyped(TextField textField, char c)
+			{
+				String proposal = employeePayField.getText();
+				int proposalValue = selectedEmployee.getHourlyWage();
+				
+				if (proposal.length() <= 0)
```

Comment:
Consider triming the proposal string before this line (proposal = proposal.trim()) to account for potential whitespace